build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Set variables for our internal Sonatype Nexus repository server
+NEXUS_USERNAME=admin
+NEXUS_PASSWORD=admin123
+NEXUS_SERVER=18.220.27.185
+NEXUS_PORT=8081
+NEXUS_REPO=repository/maven-snapshots
+NEXUS_URL=http://${NEXUS_USERNAME}:${NEXUS_PASSWORD}@${NEXUS_SERVER}:${NEXUS_PORT}/${NEXUS_REPO}
+
+# Clean, puild and deploy build artifact to internal
```

Comment:
The GAV doesn't line up with pom.xml in the cyglass-config-server project.  I'm not sure if there is a right or wrong, but either way the need to match.  In the cyglass-config-server the GAV looks like:
<groupId>org.springframework.cloud</groupId>
<artifactId>spring-cloud-config-server-cyglass</artifactId>
<version>2.2.0.BUILD-SNAPSHOT</version>

The nexus should resolve the latest snapshot build.